src/write.c

```diff
@@ -165,7 +165,7 @@ avifResult avifEncoderWrite(avifEncoder * encoder, avifImage * image, avifRWData
     avifRWStreamWriteChars(&s, "avif", 4);                         // unsigned int(32) compatible_brands[];
     avifRWStreamWriteChars(&s, "mif1", 4);                         // ... compatible_brands[]
     avifRWStreamWriteChars(&s, "miaf", 4);                         // ... compatible_brands[]
-    if ((image->depth == 8) || (image->depth == 10)) {             //
+    if (image->depth <= 10) {                                      //
```

Comment:
This isn't equivalent because the new code allows a bit depth of 9, right?

Reply:
Hi Benbuck,

This pull request modifies only the avifEncoderWrite() function. The first few lines of the function look like this:

avifResult avifEncoderWrite(avifEncoder * encoder, avifImage * image, avifRWData * output)
{
    if ((image->depth != 8) && (image->depth != 10) && (image->depth != 12)) {
        return AVIF_RESULT_UNSUPPORTED_DEPTH;
    }

Therefore, the rest of this function can assume image->depth is 8, 10, or 12.